recipes/fixit/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "fixit" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fixit-{{ version }}.tar.gz
+  sha256: b31665cb6491d659d8dfef5a6078a7e9f786e299826636d03d6bd91b6f71e95b
+
+build:
+  skip: true  # [py<38]
+  entry_points:
+    - fixit = fixit.cli:main
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - hatchling
+    - hatch-vcs
+    - pip
+  run:
+    - python
+    - click >=8.0
+    - libcst >=0.3.18
+    - moreorless >=0.4.0
+    - packaging >=21
+    - tomli >=2.0  # [py<311]
```

Comment:
It can probably be noarch if we are OK installing `tomli` in >=311. What do you think?

Reply:
I originally did that, but decided against it because it is a genuine unused dependency.

(as opposed to "try using tomli if there, otherwise use tomllib").

Reply:
We are between an unused dependency vs several artifacts that could be one and save CI time. In conda-forge we are usually biased to avoid the latter.

Reply:
If you insist, I will change this - but this policy does seem at-odds with the defaults in grayskull, which states:

> The main goal of this project is to generate concise recipes for conda-forge

Reply:
Sure. Grayskull is one thing, conda-forge is another. Also, note that grayskull has a conda-forge strict option that tries to make it closer to our latest policies, it may be outdated though.

Reply:
I generated this recipe using `mamba greyskull pypi --strict-conda-forge fixit`

Reply:
Yep, that is the option but, as I mentioned above, it is probably a bit outdated. It is hard to capture the latest policies. Well, we don't even write everything down b/c they change to quickly :-)

TL;DR ideally we should have a semi-arch for Python like we have for OS-noarch now.